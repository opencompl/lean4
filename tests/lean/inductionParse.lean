/-! should not parse `done` as part of `induction` -/

example (a : Nat) : True := by
  induction a with
  done
