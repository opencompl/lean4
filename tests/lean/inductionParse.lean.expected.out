inductionParse.lean:4:18-5:6: error: unexpected identifier; expected '|'
